chirpus/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="wrapper">
+        <h5 class="nav-link">chirpus inc.</h5>
+        <a href="#" class="nav-link">Work</a>
+        <a href="#" class="nav-link">About US</a>
+        <a href="#" class="nav-link">Contact</a>
+        <button class="button-1">Hire Us</button>
+    </div>
+    <div class="wrapper">
+        <h1>chirpus Inc.</h1>
+        <p>
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. At amet ducimus illum optio tempore minima sunt eos veritatis enim commodi, ipsam, delectus molestias doloribus quae fugit necessitatibus vitae, itaque accusamus nihil! Quisquam iure beatae commodi ad incidunt nemo consectetur deserunt! Omnis, eos nihil? Illo error aliquam et est. Recusandae, aspernatur.
+        </p>
+        <button class="button-2">call to action</button>
+    </div>
+    <footer>
+        <div class="footer">
+            <div class="footer-title">Chirpus inc.</div>
+                <div class="footer-link">
```

Comment:
This div and its closing tag should be lined up with the div on line 29. They are siblings, not parent and child.